legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/HelperRelationalBuilder.java

```diff
@@ -365,6 +362,35 @@ private static Relation findRelation(Database database, final String schemaName,
         }
     }
 
+    public static SetIterable<Table> getAllTables(Database database, SourceInformation sourceInformation)
```

Comment:
New utility methods to facilitate iterating thru all databases when querying for all tables.